.cbmc-batch/jobs/aws_cryptosdk_edk_list_clear/aws_cryptosdk_edk_list_clear_harness.c

```diff
@@ -36,8 +36,8 @@ void aws_cryptosdk_edk_list_clear_harness() {
 
     aws_cryptosdk_edk_list_clear(&edk_list);
     assert(aws_cryptosdk_edk_list_is_valid(&edk_list));
-    assert(aws_cryptosdk_edk_list_elements_are_valid(&edk_list));
 }
 
+#include <aws/common/error.inl>
```

Comment:
Why do we need commented includes bellow this line? Why do we need to keep the include at the end of the proof harness (perhaps a comment here)?

Reply:
Done